biz.aQute.resolve/src/biz/aQute/resolve/AbstractResolveContext.java

```diff
@@ -396,10 +396,11 @@ private boolean isPermitted(Resource resource) {
 			log.log(LogService.LOG_ERROR, "Resource is missing an identity capability (osgi.identity).");
 			return false;
 		}
-		if (idCaps.size() > 1) {
-			log.log(LogService.LOG_ERROR, "Resource has more than one identity capability (osgi.identity).");
-			return false;
-		}
+		// if (idCaps.size() > 1) {
```

Comment:
I know this check was added by @pkriens  for a reason but inside of a distro bundle, there are multiple osgi.identity capabilities.  So I need to be able to avoid this check or allow for it somewhow.